composer.json

```diff
@@ -65,10 +68,21 @@
         ],
         "post-install-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
-            "@drupal-scaffold"
+            "@drupal-scaffold",
+            "@kalastatic"
         ],
         "post-update-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+        ],
+        "kalastatic": [
+            "npm install",
+            "npm test"
```

Comment:
Stuck `npm test` in the `composer install` script on FTUSA because their robots weren't smart enough to run `npm test` for us. Recommend we move `npm test` after `composer test`, right after the "CALL PHPUNIT TESTS HERE".